# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-script-private.h
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/cairo-script-private.h
--- a/gfx/cairo/cairo/src/cairo-script-private.h	Mon May 06 19:22:35 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-script-private.h	Mon May 06 19:25:12 2024 +0000
@@ -52,8 +52,6 @@
 _cairo_script_context_attach_snapshots (cairo_device_t *device,
 					cairo_bool_t enable);
 
-slim_hidden_proto (cairo_script_surface_create);
-
 CAIRO_END_DECLS
 
 #endif /* CAIRO_SCRIPT_PRIVATE_H */